core/templates/dev/head/pages/library-page/search-bar/search-bar.directive.html

```diff
@@ -233,5 +226,25 @@
       margin-top: -18px;
     }
   }
-
+  @media (max-width: 740px) {
+    .oppia-search-bar {
+      right: 19px;
+      position: relative;
+    }
+    .oppia-search-bar-category-selector {
+      margin-top: 27px;
+      left: -12px;
+      position: relative;
+      display: block;
+    }
+    .oppia-same-row-container {
+      clear: both;
+      width: 250px;
+    }
```

Comment:
Add new line between the classes.

Reply:
done

Reply:
not done

Reply:
now its done, sorry for my inconvenience.